dev-tools/mage/settings.go

```diff
@@ -60,6 +60,8 @@ var (
 	XPackDir     = "../x-pack"
 	RaceDetector = false
 	TestCoverage = false
+	PLATFORMS    = EnvOr("PLATFORMS", "")
+	PACKAGES     = EnvOr("PACKAGES", "")
```

Comment:
I assume that setting this should not have any side effects in the build/packaging, so @jsoriano and @kvch can you please confirm?

Reply:
No, no side effects.